docker/README.md

```diff
@@ -75,3 +75,29 @@ Enter docker container to check any issues
 ```
 docker exec -it <container_id|container_name> /bin/bash
 ```
+
+How to Collect Docker Container Performance Metrics using OMI/OMI provider inside Docker Container
+-------
+
+OMI/OMI provider can work on containers, just make sure you know following notes:
+- OMI/OMI provider can collect containers performance metrics inside a normal container.
```

Comment:
Please clarify if "the container perf metrics" here refers to the container that OMI/OMI provider runs on or not